client/task-list/tasks/payments/components/Action.js

```diff
@@ -75,10 +76,12 @@ export const Action = ( {
 				<Button
 					className={ classes }
 					isSecondary
-					href={ getAdminLink(
-						'admin.php?page=wc-settings&tab=checkout&section=' +
-							methodKey
-					) }
+					href={ manageUrl }
+					onClick={
+						methodKey === 'wcpay'
+							? () => recordEvent( 'tasklist_payment_manage' )
```

Comment:
Seems weird we're only recording this for `wcpay`.  Should we be recording this for all methods with a prop for the method key?  cc @pmcpinto

Reply:
Good question, it sounded wcpay specific to me from the requirements, but I'll let Pedro weigh in.

Reply:
I've merged this PR, but will address this with a follow-up I currently have going depending on the response.